Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -514,11 +513,6 @@ export default function AssetPage(props: IProps) {
                     <li className="nav-item" style={{ width: '20%', paddingRight: 10 }}>
                         <fieldset className="border" style={{ padding: '10px', height: '100%' }}>
                             <legend className="w-auto" style={{ fontSize: 'large' }}>Actions:</legend>
-                            <form>
-                                <div className="form-group">
-                                    <LocationDrawings LocationID={props.Location.ID} />
-                                </div>
-                            </form>
```

Comment:
That gives us an empty `fieldset`. Looks weired

Reply:
Should I remove the section?